### DISCUSSION TITLE
Please add option for disabling mail notification and receiving them via app

### DISCUSSION THREAD

**Discussion by abh006** [community/community#3575] [Mobile]
It will be better if I can disable all the email notifications. But I dont want to miss those updates. I should still be able to see those updates in the app. So that my inbox will not be flooded with the commits and pull request updates.

**Comment by eliperkins** [comment]
Thanks for the feedback @abh006 !

You can disable emails for notifications at https://github.com/settings/notifications . Keep in mind that you'll need to leave "Web and Mobile" checked off to continue receiving notifications within the mobile apps.

**Reply by nobourge** [reply]
can I be in-app- & email- notified for a repository & only in-app for another?

**Comment by osamashadwo** [comment]
Send me option ple

**Comment by ghost** [comment]
> Thanks for the feedback @abh006 !
> 
> You can disable emails for notifications at https://github.com/settings/notifications . Keep in mind that you'll need to leave "Web and Mobile" checked off to continue receiving notifications within the mobile apps.
